### DISCUSSION TITLE
Hello I have a problem pls help When I do and upgrade the island upgrade and then when I reconnect it reset

### DISCUSSION THREAD

**Discussion by CRAFTINGHARD1234** [BG-Software-LLC/SuperiorSkyblock2#367] [Denied Feature Requests] [answered]
Hello I have a problem pls help
When I do and upgrade the island upgrade and then when I reconnect it reset

**Comment by OmerBenGera** [comment, marked as answer]
Go to the discord and ask for help, this is really not the place to ask for support.
